src/controllers/index.ts

```diff
@@ -1,5 +1,47 @@
+import { Request, Response } from 'express';
+import {OrmConfig} from '../configs/db_config';
+import { User } from '../entity/User';
+import bcrypt from 'bcrypt';
 // export all controllers
 function myFunction (): void {
   console.log('Hello');
 }
 myFunction();
```

Comment:
If this function is doing nothing i thing it's better if you delete it

Reply:
@maxCastro1 !
Look into this as well !

Reply:
noted